app/controllers/concerns/idv/step_indicator_concern.rb

```diff
@@ -2,8 +2,6 @@ module Idv
   module StepIndicatorConcern
     extend ActiveSupport::Concern
 
-    include IdvSession
```

Comment:
Are we sure that the before actions in IdvSession weren't needed in some of the many places the StepIndicator is used? Seems like the enclosing controller should also include it, but it's worth checking.
```
    before_action :redirect_unless_effective_user
    before_action :redirect_if_sp_context_needed
```

Reply:
Hmm, let me do a quick check and sprinkle some `include IdvSession` lines around

Reply:
I did a search for `include StepIndicatorConcern` and found 2 controllers that didn't have `IdvSession` included via some other means. Updated in [46429cd](https://github.com/18F/identity-idp/pull/8480/commits/46429cd8f9a80d99991e2854dc7b01615b8f62a1)